cdk/bin/amiable.ts

```diff
@@ -4,18 +4,21 @@ import { App } from "aws-cdk-lib";
 import { Amiable } from "../lib/amiable/amiable";
 
 const app = new App();
-new Amiable(app, "Amiable-CODE", {
-  migratedFromCloudFormation: true,
-  stack: "deploy",
+
+const commonProps = { migratedFromCloudFormation: true, stack: "deploy", env: { region: "eu-west-1" } };
+
+export const codeProps = {
```

Comment:
Using `const`s here allows us to pass identical props into the tests. This prevents duplication and makes the snapshot tests as close as possible to the actual templates that will be deployed.